src/pages/iou/request/step/IOURequestStepDestination.tsx

```diff
@@ -112,56 +115,62 @@ function IOURequestStepDestination({
     };
 
     return (
-        <StepScreenWrapper
-            headerTitle={backTo ? translate('common.destination') : tabTitles[iouType]}
-            onBackButtonPress={navigateBack}
-            shouldShowWrapper={!openedFromStartPage}
-            shouldShowNotFoundPage={shouldShowNotFoundPage}
-            testID={IOURequestStepDestination.displayName}
+        <ScreenWrapper
+            includePaddingTop={false}
+            keyboardVerticalOffset={variables.contentHeaderHeight + top + variables.tabSelectorButtonHeight + variables.tabSelectorButtonPadding}
+            testID={`${IOURequestStepDestination.displayName}-container`}
```

Comment:
Coming from this issue https://github.com/Expensify/App/issues/72833, we’re wrapping the `StepScreenWrapper` with `ScreenWrapper`. However, since `StepScreenWrapper` has its own offline indicator and `ScreenWrapper` does too, the offline indicator was appearing twice. We’ve fixed that here: https://github.com/Expensify/App/issues/72833#issuecomment-3413863292